test/helpers/ghostferry_helper.rb

```diff
@@ -49,12 +51,8 @@ module Status
 
     attr_reader :stdout, :stderr, :logrus_lines, :exit_status, :pid, :error, :error_lines
 
-    def initialize(main_path, config: {}, logger: nil, message_timeout: 30, port: 39393)
+    def initialize(main_path, config: {}, logger:, message_timeout: 30, port: 39393)
```

Comment:
>what is the benefit of requiring this in a test helper?
Because then we don't have to lazy initialize it, and we always set it up anyway

> do we want to update the comments under the class to highlight that the logger is now required?
Can you point me to those? I can't seem to find any